app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -42,6 +42,24 @@
     LoginButton loginButton;
     CallbackManager callbackManager;
     Activity context;
+    boolean onSuccessCalled;
+    public static final String NAME = "name";
```

Comment:
Lets make all these strings private since we are not using these outside this class